### DISCUSSION TITLE
Skin Editor should have same aspect ratio

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#17553] [Ideas] [answered]
Version 2022.327.0
Because the aspect ratio changes when moving from the skin editor to gameplay, skin elements are difficult to align with the taiko playfield. For example, in these screenshots, I try to roughly center the blue section in the UR bar with the taiko playfield:
![osu_2022-03-30_00-35-02](https://user-images.githubusercontent.com/74262407/160777663-02235b3d-c127-4767-b5d4-6e1493b2f30a.png)
![osu_2022-03-30_00-35-09](https://user-images.githubusercontent.com/74262407/160777669-14edc553-ad9b-440d-86b8-4849c998a681.png)
![osu_2022-03-30_00-35-45](https://user-images.githubusercontent.com/74262407/160777671-803ad0f4-9c20-47c1-b38e-450b2e62c4ff.png)
![osu_2022-03-30_00-35-48](https://user-images.githubusercontent.com/74262407/160777672-98d2f8e0-6177-4e03-89be-cdfa12c385c4.png)

**Comment by peppy** [comment, marked as answer]
Okay so this is a warning. You are a regular contributor so you should know better. If you aren't able to search I may restrict your posting abilities to reduce noise.

https://github.com/ppy/osu/discussions/17407

![Safari 2022-03-30 at 07 42 58](https://user-images.githubusercontent.com/191335/160778409-56f269c5-8357-4fe6-9e94-01905a7218c7.png)

**Reply by zachmanthethird** [reply]
Oh GitHub isn't sorting by date properly... my bad. :confused:
![image](https://user-images.githubusercontent.com/74262407/160779884-248d04b2-1648-4b56-9310-4fc762ca4483.png)
